inference-engine/src/vpu/graph_transformer/src/stages/interpolate.cpp

```diff
@@ -39,78 +60,79 @@ void FrontEnd::parseInterpolate(const Model& model, const ie::CNNLayerPtr& _laye
                 // coordinate_transformation_mode = asymmetric; nearest_mode = floor;
                 // other "Interpolate" modes are translated to the default ones
-                const auto antialias = _layer->GetParamAsBool("antialias", false);
-                const auto coordinateTransformation = _layer->GetParamAsString("coordinate_transformation_mode", "half_pixel");
-                const auto nearest = _layer->GetParamAsString("nearest_mode", "round_prefer_floor");
+                const auto anti = _layer->GetParamAsBool(g_antialias, false);
+                const auto coordinateTransformation = _layer->GetParamAsString(g_coordinate_transformation_mode, g_half_pixel);
+                const auto near = _layer->GetParamAsString(g_nearest_mode, g_round_prefer_floor);
                 InterpolateCoordTransMode coordinateTransformationMode = InterpolateCoordTransMode::HalfPixel;
                 InterpolateNearestMode nearestMode = InterpolateNearestMode::RoundPreferCeil;
 
-                if (cmp(coordinateTransformation, "asymmetric")) {
-                    coordinateTransformationMode = InterpolateCoordTransMode::Asymmetric;
-                } else if (cmp(coordinateTransformation, "half_pixel")) {
-                    coordinateTransformationMode = InterpolateCoordTransMode::HalfPixel;
-                } else if (cmp(coordinateTransformation, "pytorch_half_pixel")) {
-                    coordinateTransformationMode = InterpolateCoordTransMode::PytorchHalfPixel;
-                } else if (cmp(coordinateTransformation, "tf_half_pixel_for_nn")) {
-                    coordinateTransformationMode = InterpolateCoordTransMode::TfHalfPixelForNn;
-                } else if (cmp(coordinateTransformation, "align_corners")) {
-                    coordinateTransformationMode = InterpolateCoordTransMode::AlignCorners;
-                }
-
-                if (cmp(nearest, "round_prefer_floor")) {
+                if (cmp(near, g_round_prefer_floor)) {
                     nearestMode = InterpolateNearestMode::RoundPreferFloor;
-                } else if (cmp(nearest, "round_prefer_ceil")) {
+                } else if (cmp(near, g_round_prefer_ceil)) {
                     nearestMode = InterpolateNearestMode::RoundPreferCeil;
-                } else if (cmp(nearest, "floor")) {
+                } else if (cmp(near, g_floor_mode)) {
                     nearestMode = InterpolateNearestMode::Floor;
-                } else if (cmp(nearest, "ceil")) {
+                } else if (cmp(near, g_ceil_mode)) {
                     nearestMode = InterpolateNearestMode::Ceil;
-                } else if (cmp(nearest, "simple")) {
+                } else if (cmp(near, g_simple)) {
                     nearestMode = InterpolateNearestMode::Simple;
+                } else {
+                    VPU_THROW_FORMAT("Current Interpolate does not support this nearest mode");
                 }
```

Comment:
Nitpick: I think that should be extracted to a function as well, even if it's used only in one place

Reply:
Probably it should be located at the same place as global constants along with `InterpolateCoordTransMode ` and `InterpolateNearestMode `